### ISSUE TITLE
Error: Cannot find module '@nestjs/core/router/route-path-factory' on @nestjs/swagger : "^5.0.0"

### ISSUE THREAD

**Issue by hussainwali74** [nestjs/swagger#1445]
## I'm submitting a...

<pre><code>
[ ] Regression 
[ ] Bug report
[ ] Feature request
[*] Documentation issue or request
[ ] Support request => Please do not submit support request here, instead post your question on Stack Overflow.
</code></pre>

## Current behavior

my package.json
  ```
"dependencies": {
    "@nestjs/common": "^7.6.15",
    "@nestjs/mapped-types": "*",
    "@nestjs/passport": "^8.0.0",
    "@nestjs/core": "^7.6.15",
    "@nestjs/platform-express": "^7.6.15",
    "@nestjs/swagger": "^5.0.0",
    "@nestjs/typeorm": "^8.0.1",
    "class-transformer": "^0.4.0",
    "class-validator": "^0.13.1",
    "mysql2": "^2.2.5",
    "passport": "^0.4.1",
    "passport-local": "^1.0.0",
    "reflect-metadata": "^0.1.13",
    "rimraf": "^3.0.2",
    "rxjs": "^7.0.0",
    "swagger-ui-express": "^4.1.6",
    "typeorm": "^0.2.34"
  },
```

getting this error on a new installation of nest and swagger:

```
internal/modules/cjs/loader.js:883
  throw err;
  ^

Error: Cannot find module '@nestjs/core/router/route-path-factory'
Require stack:
- D:\thesis\App\backend\node_modules\@nestjs\swagger\dist\swagger-explorer.js  
- D:\thesis\App\backend\node_modules\@nestjs\swagger\dist\swagger-scanner.js   
- D:\thesis\App\backend\node_modules\@nestjs\swagger\dist\swagger-module.js    
- D:\thesis\App\backend\node_modules\@nestjs\swagger\dist\index.js
- D:\thesis\App\backend\node_modules\@nestjs\swagger\index.js
- D:\thesis\App\backend\dist\main.js
    at Function.Module._resolveFilename (internal/modules/cjs/loader.js:880:15)
    at Function.Module._load (internal/modules/cjs/loader.js:725:27)
    at Module.require (internal/modules/cjs/loader.js:952:19)
    at require (internal/modules/cjs/helpers.js:88:18)
    at Object.<anonymous> (D:\thesis\App\backend\node_modules\@nestjs\swagger\dist\swagger-explorer.js:8:30)
    at Module._compile (internal/modules/cjs/loader.js:1063:30)
    at Object.Module._extensions..js (internal/modules/cjs/loader.js:1092:10)     
    at Module.load (internal/modules/cjs/loader.js:928:32)
    at Function.Module._load (internal/modules/cjs/loader.js:769:14)
    at Module.require (internal/modules/cjs/loader.js:952:19) {
  code: 'MODULE_NOT_FOUND',
  requireStack: [
    'D:\\thesis\\App\\backend\\node_modules\\@nestjs\\swagger\\dist\\swagger-explorer.js',
    'D:\\thesis\\App\\backend\\node_modules\\@nestjs\\swagger\\dist\\swagger-scanner.js',
    'D:\\thesis\\App\\backend\\node_modules\\@nestjs\\swagger\\dist\\swagger-module.js',
    'D:\\thesis\\App\\backend\\node_modules\\@nestjs\\swagger\\dist\\index.js',   
    'D:\\thesis\\App\\backend\\node_modules\\@nestjs\\swagger\\index.js',
    'D:\\thesis\\App\\backend\\dist\\main.js'
  ]
}

```

## Minimal reproduction of the problem with instructions

steps to reproduce :
1. new installation of latest NestJs
2. install swagger using `npm install --save @nestjs/swagger swagger-ui-express`
3. npm run start:dev

**Comment by jmcdo29** [comment]
[A simple search in the closed issues shows this has already been discussed and resolved.](https://github.com/nestjs/swagger/issues/1433)

**Comment by hussainwali74** [comment]
for anyone else searching for fix for this issue, you can use this as a **last resort**:
`nest update -f -t latest`

**Comment by SchroederSteffen** [comment]
As I can't comment on https://github.com/nestjs/swagger/issues/1449 because it's closed, asking here:
Shouldn't the [package.json](https://github.com/nestjs/swagger/blob/833458d371d929d14ed5d9b8c32f51429e56dd28/package.json#L59) state that @nestjs/core v8 is required for @nestjs/swagger v5?

**Comment by kamilmysliwiec** [comment]
Great catch! Fixed in 5.0.9
